src/theory/main.tex

```diff
@@ -47,7 +48,9 @@ \subsection{Structure from Motion}\label{sec:sfm}
     \label{fig:theory-sfm-parallax}
 \end{figure}
 
-Um die Kameraposition sowie die Geometrie der Szene simultan zu rekonstruieren werden automatisch Feature in den Bildern identifiziert. Diese Feature werden dann über mehrere Bilder verfolgt was eine erste Annahme zu den Kamerapositionen sowie den Objektkoordinaten erlaubt welche iterativ mit nicht-linearer least-squares minimisation verfeinert werden können. ~\cite[Kapitel 1.3]{westoby_2012}
+Um die Kameraposition sowie die Geometrie der Szene simultan zu rekonstruieren werden automatisch Feature in den Bildern identifiziert.
+Diese Feature werden dann über mehrere Bilder verfolgt, was eine erste Annahme zu den Kamerapositionen sowie den Objektkoordinaten erlaubt.
```

Comment:
Mit diesen Features lässt sich die Kameraposition und ihre Ausrichtung über mehrere Bilder hinweg bestimmen.
Somit können die Bildpunkte der Features, wie in Abschnitt XXX beschrieben, rekonstruiert werden. (Kapitel zur Triangulation kommt noch)